src/Reflection/ReflectionType.php

```diff
@@ -52,6 +52,11 @@ public static function createFromNode(
         return new ReflectionUnionType($reflector, $owner, $type);
     }
 
+    public function getOwner(): ReflectionParameter|ReflectionMethod|ReflectionFunction|ReflectionEnum|ReflectionProperty
```

Comment:
Should we perhaps mark this `@internal`, so we don't necessarily need to make this public API (yet)?

Reply:
👍 Added